docs/modules/globby.md

```diff
@@ -0,0 +1,32 @@
+# globby
+
+`globby` is a globbing library that presents [some behaviors](https://github.com/sindresorhus/globby#features)
+no other libraries have, mainly the handling of negated patterns and directory expansion.
+
+It pulls in [20+](https://npmgraph.js.org/?q=globby) transitive dependencies and can be replaced
```

Comment:
I wouldn't mention the number of dependencies as it can change. its better to just mention there are lighter alternatives (that's what we do in other docs too)